float32[] cmd
float32 duration
---
int16 result		# 1:success, -1:false